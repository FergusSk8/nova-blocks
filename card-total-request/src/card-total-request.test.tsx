import React from 'react';
import { shallow } from 'enzyme';
import TotalRequestCard from './index'


describe('TotalRequestCard', () => {
    let wrapper = shallow(<TotalRequestCard />);

    test('should show correctly', () => {
      expect(wrapper).toMatchSnapshot();
    });
  
});