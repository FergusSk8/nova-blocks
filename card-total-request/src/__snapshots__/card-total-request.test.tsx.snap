// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TotalRequestCard should show correctly 1`] = `
<Card
  bodyStyle={
    Object {
      "padding": "20px",
    }
  }
>
  <div>
    <div
      className=""
    >
      <div>
        <div>
          <span>
            Total de solicitudes
          </span>
          <span />
        </div>
        <div>
          <div>
            0
          </div>
          <div>
            <img
              src="test-file-stub"
            />
          </div>
        </div>
      </div>
    </div>
    <div>
      Estatus de solicitudes
    </div>
    <div>
      <div>
        <p>
          0
          <span>
            Iniciadas
          </span>
        </p>
        <div>
          <img
            src="test-file-stub"
          />
        </div>
      </div>
      <div>
        <p>
          0
          <span>
            En proceso
          </span>
        </p>
        <div>
          <img
            src="test-file-stub"
          />
        </div>
      </div>
      <div>
        <p>
          0
          <span>
            Abandonadas
          </span>
        </p>
        <div>
          <img
            src="test-file-stub"
          />
        </div>
      </div>
      <div>
        <p>
          0
          <span>
            Terminadas
          </span>
        </p>
        <div>
          <img
            src="test-file-stub"
          />
        </div>
      </div>
    </div>
  </div>
</Card>
`;
